index.js

```diff
@@ -82,6 +82,14 @@ module.exports = (input, opts) => {
 			})
 		},
 		{
+			title: 'Pre-release version check',
+			task: () => {
+				if ((VERSIONS.slice(3).includes(input) || semver.prerelease(input)) && !opts.tag) {
```

Comment:
You can't use `.includes()`. This project still supports Node.js 4. Use `.indexOf` instead.